Formula/wasi-sdk.rb

```diff
@@ -0,0 +1,55 @@
+class WasiSdk < Formula
+  desc "WASI-enabled WebAssembly C/C++ toolchain"
+  homepage "https://github.com/WebAssembly/wasi-sdk"
+  url "https://github.com/WebAssembly/wasi-sdk.git",
+    tag:      "wasi-sdk-15",
+    revision: "e025da6b1099c730af4809484ad0ea84e0c3edbe"
+  version "15.0"
+  license "Apache-2.0"
+
+  keg_only "provides alternative LLVM/Clang binaries specific to WASI"
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "python@3.10" => :build
+  depends_on "rsync" => :build
```

Comment:
In a local test I got this to build once I made it use brewed GCC. So we just need to add:
```
on_linux do
   depends_on "gcc"
end

fails_with gcc: "5"
```

Reply:
Thanks! I've incorporated this change now.